lms/djangoapps/certificates/api.py

```diff
@@ -425,51 +396,66 @@ def example_certificates_status(course_key):
     return ExampleCertificateSet.latest_status(course_key)
 
 
+def _safe_course_key(course_key):
+    if not isinstance(course_key, CourseKey):
+        try:
+            return CourseKey.from_string(course_key)
+        except InvalidKeyError:
+            log.warning('Unable to parse course_key "%s"', course_key, exc_info=True)
+            return
```

Comment:
So what happens when this line gets hit? Is this equivalent to `return None`?

Reply:
Yup, `return` and `return None` are synonyms.